tools/opt/opt.cpp

```diff
@@ -23,10 +25,22 @@
 #include "spirv-tools/optimizer.hpp"
 
 #include "message.h"
+#include "opt-passes.h"
+#include "size-passes.h"
 #include "tools/io.h"
 
 using namespace spvtools;
 
+std::string Flags2Str(const std::vector<std::string>& flags) {
```

Comment:
Need a method comment.

Also, I prefer To to 2.

Reply:
Done.